src/mail/ngx_mail_auth_http_module.c

```diff
@@ -1133,6 +1133,7 @@ ngx_mail_auth_http_create_request(ngx_mail_session_t *s, ngx_pool_t *pool,
     ngx_mail_auth_http_conf_t *ahcf)
 {
     size_t                     len;
+    ngx_uint_t                 client_port;
     ngx_buf_t                 *b;
     ngx_str_t                  login, passwd;
     ngx_connection_t          *c;
```

Comment:
Seeing as we use 'Christmas Tree' ordering for variable declarations, the `ngx_uint_t` variable should come between `ngx_connection_t` and `ngx_str_t`.

Thanks.

Reply:
@ac000 Thanks, I fixed it